internal/docstore/driver/driver.go

```diff
@@ -71,3 +83,28 @@ type Mod struct {
 	FieldPath []string
 	Value     interface{}
 }
+
+// An ActionListError contains all the errors encountered from a call to RunActions,
+// and the positions of the corresponding actions.
+type ActionListError []struct {
```

Comment:
Maybe `ActionListErrors` or `ActionErrorList`?

Reply:
I think singular (ActionListError) makes a little more sense. It is one `error`, even though it holds multiple things. For comparison, Hashicorp's version of this is called [multierror](https://github.com/hashicorp/go-multierror).